bitnami/tomcat/templates/_helpers.tpl

```diff
@@ -28,6 +28,18 @@ If release name contains chart name it will be used as a full name.
 {{- include "common.names.fullname" . -}}
 {{- end -}}
 
+{{/*
+Return the appropriate apiVersion for networkpolicy.
+*/}}
+{{- define "tomcat.networkPolicy.apiVersion" -}}
```

Comment:
should this macro be moved into bitnami/common ?

Reply:
Hi, 
Yes it should be. I have update NP to used common capabilities. I have based my update on pg, i update PG after this PR.
thanks,